routes/notes.js

```diff
@@ -0,0 +1,16 @@
+const express = require("express");
+const router = express.Router();
+const protected = require("../middleware");
+const helpers = require("../helpers");
```

Comment:
This part (register and login also) could be improved a little bit by using ES6 destructuring. So instead You could do something like this to make code a little more clean:

```
const {
  getNotes, addNote, getNote, editNote, deleteNote } = require("../helpers");

router
  .route("/")
  .get(protected, getNotes)
  .post(protected, addNote);
 etc...
```